Apple page

dannkkie opened the pull request:
Solo project done please review

dannkkie committed:
Add images directory

Added images directory with the images needed for the page

dannkkie committed:
Add index.html to root

dannkkie committed:
Add css directory to root

Added a css directory to root containing styles.css file to be used in index.html in the root for styling the page

dannkkie committed:
Add css styling

Added the css styles for each of the divs and tags

dannkkie committed:
Edit css style

dannkkie committed:
Edit index.html

dannkkie committed:
Add nav bar

Added the nav bar and the links with gradient background

dannkkie committed:
Style the search input

dannkkie committed:
rename abbreviated class nad id names to descriptive names

dannkkie committed:
rename images with descriptive names

dannkkie committed:
remove unused images from images folder

dannkkie committed:
Edit README and add screenshot image

Edited README file and added screenshot of project page

Forison reviewed (commented):
# Pull request under review by Addo

dannkkie commented:
Okay

Forison reviewed (changes requested):
# Hi @Edmund👏 
You have done a good job so far.🙂
This is the output I get from your project🙂
![Screenshot from 2019-11-05 18-31-09](https://user-images.githubusercontent.com/29695661/68255601-ac0e4000-fffb-11e9-9752-a1965baa6afa.png)
![Screenshot from 2019-11-05 18-31-14](https://user-images.githubusercontent.com/29695661/68255604-aca6d680-fffb-11e9-9a8a-e2b22904b11f.png)

#### Here are a few changes we have to make to get this project approved🙂
- [x] It should be made of nine different buttons with the Apple logo on the leftmost section and a search field on the rightmost section🙂
Let us replace this ```<a href="#"><div class="store"></div></a>``` with ```<a href="#">store</a>``` and so on
- [x] The main content should have an image set as background (as CSS [`background-image`]
currently it is blank🙂

## That is it Edmund, Kindly do these changes and submit for view🙂
feel free to contact me on [LinkedIn](https://www.linkedin.com/in/forison-addo-558201159/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. I would be happy to assist you.
#### Happy coding🙂

dannkkie commented:
okay doing that now

dannkkie commented:
For the hero image you are not seeing i doubt if it is from me because everything is showing well here

Redvanisation commented on css/styles.css:
```diff
@@ -0,0 +1,287 @@
+}
+
+#nav-links {
+  background-image: url("/images/nav-surround.png");
```
The hero image is not showing, try this instead `url("../images/nav-surround.png")`

Redvanisation reviewed (changes requested):
# Changes required  🛑

### Hello @dannkkie   🌟

**Good work implementing the changes, but we need to fix the issues below before moving on ⬇️** 

1. The nav buttons don't have text, please have the nav looking like the original page.

2. The hero background image is not showing, check my comment below on how to fix that.

3. the 4 bottom elements under the image should be links.

4. The footer link aren't aligned, please align them like on the original page.

### _After making the required changes please submit a new code review request_

### Good luck guys! 🍀

dannkkie committed:
Resize header conatiner

Resized the header container to fill more space and also increased the hero image to correspond with it.

dannkkie committed:
Change nav bar to use text

Changed the nav bar to use text instead of images in the menu names

dannkkie commented:
Please check and review kindly check your browser because the footer is well arranged just like in the original page and the 4 images are links you can inspect the code. All other changes have been made accordingly

dannkkie committed:
change top value on line 21 to 70px

ebukaume commented:
## Changes Required! :stop_sign: 

Hi @dannkkie :wave: 

I recommend you consider @Redvanisation suggestion.
> The hero image is not showing, try this instead url("../images/nav-surround.png")

I see this when I view your page using raw githack as well as when using my local file system.
![review-apple-dan](https://user-images.githubusercontent.com/22497270/68324968-c82be300-00c8-11ea-9367-7b8e7202048c.png)

However, when I spin up live-server, I get this :point_down:  (not taking full width)
![review-apple-dan2](https://user-images.githubusercontent.com/22497270/68325001-e0036700-00c8-11ea-887e-184857f088ac.png)

Please refer to @Redvanisation's last review and make the requested fixes. Your images (4) are not behaving like links _(clickable, pointer cursor, etc)_. Kindly fix those too.

**Submit another code review request after making those changes**


Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

dannkkie commented:
The live page we are supposed to clone doesn't take full width also i have made changes to this url("../images/nav-surround.png") as directed by @Redvanisation

dannkkie committed:
chnaged the z-index value for aside tag for images to be clickable

shshamim63 commented:
### :arrow_forward:  Status Approved!! :arrow_backward: 
___
Hi @dannkkie :wave: ,
All the part that are required to declare this project are in here. Nice job. :clap: 
#### This :+1:  is from me, you can merge this and don't forget to fill the form.
___
### Happy Coding :herb: 
#### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

dannkkie commented:
Thank you @shshamim63
